lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -24,11 +24,18 @@ def execute(self) -> int:
             self.config,
             self.logger,
         )
-        crt.copy_result_tb(
+        res = crt.copy_result_tb(
             self.context.token, self.context.access, self.context.metadata
         )
-        return 0
-
+        if 200 <= res.status_code <= 201:
+            click.echo(f"Status: {res.status_code}, reason: {res.reason}, message: {res.message}")
```

Comment:
I'm not sure that breaking out the three values this way makes sense. `reason` is just the canonical description of `status_code`: `200/OK`, `400/Bad request`; `message` is the raw response from the HTTP server, which (in the case of our server) is a JSON document containing a `"message"` field with more detailed information. I might even be inclined to use that only as a fallback if there's no `response.data`, starting with `response.json["message"]`, falling back to `response.text` if it doesn't jsonify (e.g., if the content type isn't JSON), and finally to `response.reason`.